boards/st/nucleo_l552ze_q/Kconfig.nucleo_l552ze_q

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 Linaro Limited
+# SPDX-License-Identifier: Apache-2.0
+
+config BOARD_NUCLEO_L552ZE_Q
+	select SOC_STM32L552XX
+
+config BOARD_NUCLEO_L552ZE_Q_STM32L552XX_NS
+	bool
+	select SOC_STM32L552XX
```

Comment:
This seems shaky as when building ns target `west build -b  nucleo_l552ze_q/stm32l552xx/ns`, both symbols ends up being defined:
```
$ grep -r BOARD_NUCLEO_L552ZE_Q build/nucleo_l552ze_q/stm32l552xx/ns/zephyr/.config 
CONFIG_BOARD_NUCLEO_L552ZE_Q=y
CONFIG_BOARD_NUCLEO_L552ZE_Q_STM32L552XX_NS=y

```

Reply:
@tejlmand, @nordicjm. I would welcome advices

Reply:
This is expected, `BOARD_NUCLEO_L552ZE_Q` is the board itself and `BOARD_NUCLEO_L552ZE_Q_STM32L552XX_NS` is the variant

Reply:
ok, I'll keep it that way then. Txs